Xamarin.PropertyEditing/Properties/Resources.resx

```diff
@@ -554,4 +554,16 @@
   <data name="Shared" xml:space="preserve">
     <value>Shared</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="AddVariationHelpText" xml:space="preserve">
+    <value>Introduce variation based on:</value>
+  </data>
+  <data name="AddVariationTitle" xml:space="preserve">
+    <value>Add a Variation</value>
+  </data>
+  <data name="Any" xml:space="preserve">
+    <value>Any</value>
+  </data>
+  <data name="ErrorCreatingVariant" xml:space="preserve">
+    <value>Error creating variant: {0}. If this error continues, please report it.</value>
```

Comment:
That may vary depending on which product this is used in. We might need to come up with something different.

Reply:
Maybe [here](https://github.com/xamarin/Xamarin.PropertyEditing/issues) is a good start.

Reply:
It'd be an issue raised from the provider though, not us.

Reply:
I guess I'm not exactly sure who is most likely to read this message, and thus where reporting it would be most appropriate. If we can't tell, maybe we shouldn't even suggest reporting it, and instead just state the error.